docker-compose-fabric.yaml

```diff
@@ -36,7 +36,7 @@ services:
       - ./examples/apps/generic_client/:/project/avalon/examples/apps/generic_client/
       - ~/.Xauthority:/root/.Xauthority
       - ~/mywork/vars/keyfiles:/keyfiles
-      - ~/mywork/vars/mychannel_network.json:/project/avalon/sdk/avalon_sdk/fabric/network.json
+      - ~/mywork/vars/profiles/mychannel_network_for_pysdk.json:/project/avalon/sdk/avalon_sdk/fabric/network.json
```

Comment:
How is the file "mychannel_network_for_pysdk.json" getting created ?

Reply:
It is generated from minifab in work_dir(~/mywork/var

Reply:
Please add a comment in docker-compose file and/or startup script